api_client.py

```diff
@@ -281,6 +282,7 @@ def get_alerts_or_events_req_args(self, params):
             )
         else:
             args = "&".join(["%s=%s" % (k, v) for k, v in params.items()])
+        args+='&from_date_offset_minutes='+str(self.config.events_from_date_offset_minutes if endpoint_name=="events" else self.config.alerts_from_date_offset_minutes)
```

Comment:
Looks like some kinda indentation problem. Add spaces around operators. Don't use trailing "if ... else ...". It complicates the code, unnecessarily.

Reply:
Done